build.gradle

```diff
@@ -2,7 +2,7 @@ import edu.wpi.first.gradlerio.deploy.roborio.RoboRIO
 
 plugins {
     id "java"
-    id "edu.wpi.first.GradleRIO" version "2022.1.1"
+    id "edu.wpi.first.GradleRIO" version "2022.2.1"

```

Comment:
Make sure that this matches the version on the driver station—probably leave it as `2022.1.1` for now.

Reply:
Don't worry about changing this at the moment, though.